File: Examples/01_Simple/Simple/ViewController.m
//
//  ViewController.m
//  SimpleApp
//
//  Created by Tom Yu on 2020/10/12.
//

#import "ViewController.h"
#import <SimplePod/Cat.h>
#import <SimplePod/Dog.h>
#import <SimplePod/Rose.h>
#import <SimplePod/AppleTree.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [[[Cat alloc] init] run];
    [[[Dog alloc] init] eat];
    [[[Rose alloc] init] bloom];
    [[[AppleTree alloc] init] fruit];
}

@end

File: Examples/01_Simple/Simple/ViewController.swift
//
//  ViewController.swift
//  Simple
//
//  Created by Tom Yu on 2025/3/20.
//

import UIKit
import SimplePod

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        Cat().run();
        Dog().eat();
        Rose().bloom();
        AppleTree().fruit();
    }
}

